examples/developer-tools/src/storage.ts

```diff
@@ -4,12 +4,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+const settingsKey = 'blockFactorySettings';
```

Comment:
Is this a constant? https://google.github.io/styleguide/tsguide.html#identifiers-constants

Reply:
Yeah, but our samples style guide prohibits CONSTANT_CASE haha.

I think Aaron was looking at fixing this recently so I'll check in on that, but will leave this for now